apps/health_campaign_field_worker_app/lib/models/downsync/downsync.mapper.dart

```diff
@@ -14,6 +14,7 @@ class DownsyncSearchModelMapper
   static DownsyncSearchModelMapper ensureInitialized() {
     if (_instance == null) {
       MapperContainer.globals.use(_instance = DownsyncSearchModelMapper._());
+      EntitySearchModelMapper.ensureInitialized().addSubMapper(_instance!);
```

Comment:
Cross check the dart_mappable package. It should be from local. If it's is local this will not be generated.

Reply:
> Cross check the dart_mappable package. It should be from local. If it's is local this will not be generated.

yes I made the local import now, so now its not being generated